### ISSUE TITLE
Android App stuck in "Wait" screen

### ISSUE THREAD

**Issue by TheKeyLimePie** [haiwen/seadroid#1045]
With version 3.0.6 of the Android app I'm unable to open the Seafile overview. When opening the app the view is stuck in a "Wait" screen with a spinning loading indicator.
![Screenshot_20241224-100828](https://github.com/user-attachments/assets/568cba21-7d8b-4901-aab4-26ca71850036)
The web interface of my Seafile instance (v 11.0.13) is working fine.
Force-quitting the app and clearing the app's cache did not solve the problem.

**Comment by zhwanng** [comment]
"Wait" indicates that the app is migrating data, it usually ends quickly, Is your "wait" not over for a long time?

**Comment by TheKeyLimePie** [comment]
Hello zhwanng,

well it seems I was just too impatient. I gave it another try and kept the app open in foreground for about 3 minutes and let it do whatever it had to do. Now it's finished and launches like usual.
Device used: Google Pixel 7 Pro on Android 15.

I haven't experienced such a long migration on Seafile before and it feels a little bit odd to me for an app to take that much time. Maybe a progress bar or status text would help to indicate heavy migrations in the future...

Anyway, this issue can be closed as it was due to my impatience.
